cmd/probercmd.go

```diff
@@ -18,7 +18,7 @@ import (
 	"flag"
 	"fmt"
 
-	"github.com/gardener/dependency-watchdog/controllers"
+	"github.com/gardener/dependency-watchdog/controllers/cluster"
 	"github.com/gardener/dependency-watchdog/internal/prober"
 	"github.com/gardener/dependency-watchdog/internal/util"
 	gardenextensions "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
```

Comment:
call it `extensionsv1alpha1` or `gardenerextensionsv1alpha1`

Reply:
done